EpicCactus94.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.
+
+## Plan
+
+I've already begun designing the .stl files which will be used to print the cube, and I've assured access to 3D printers before the building week. 

```

Comment:
Are you using CAD tools to design the STLs? If so, can you describe that here in your document?

Reply:
I used StoneWorks to design the STL files. It wasn't too complex because it is just the frame of the cube which will contain all of the necessary parts. They will also include a place to actually fix the motors and arduino onto the cube so the stuff won't fall out when the cube is flipped.

Reply:
Cool! Could you add this to your markdown document?